javascript-carousel/main.js

```diff
@@ -0,0 +1,255 @@
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "fas fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($charmander.className === "charmander") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "fas fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($squirtle.className === "squirtle") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "fas fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  } else if ($pikachu.className === "pikachu") {
+    $bulbasaur.className = "bulbasaur hidden";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff"
+    $circleOne.className = "far fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "fas fa-circle circle-btn circle-btn-five";
+  } else if ($jigglypuff.className === "jigglypuff") {
+    $bulbasaur.className = "bulbasaur";
+    $charmander.className = "charmander hidden";
+    $squirtle.className = "squirtle hidden";
+    $pikachu.className = "pikachu hidden";
+    $jigglypuff.className = "jigglypuff hidden"
+    $circleOne.className = "fas fa-circle circle-btn circle-btn-one";
+    $circleTwo.className = "far fa-circle circle-btn circle-btn-two";
+    $circleThree.className = "far fa-circle circle-btn circle-btn-three";
+    $circleFour.className = "far fa-circle circle-btn circle-btn-four";
+    $circleFive.className = "far fa-circle circle-btn circle-btn-five";
+  }
```

Comment:
Same goes for here, you can iterate through a loop of dom elements, and verify what should be on the dom using a variable in your code.

Reply:
Updated per the feedback above, new gif is below.

![20201210165952115](https://user-images.githubusercontent.com/69181727/101848266-e7861280-3b09-11eb-8c90-d9e1520cd6b0.gif)